src/main/helm/helm-release-manager.ts

```diff
@@ -179,12 +179,13 @@ export async function getHistory(name: string, namespace: string, kubeconfigPath
 }
 
 export async function rollback(name: string, namespace: string, revision: number, kubeconfigPath: string) {
-  return JSON.parse(await execHelm([
+  await execHelm([
```

Comment:
Result is not used. And actually it has never worked. Since `execHelm` returns `string` and `JSON.parse("some string")` throws